src/ray/util/logging.cc

```diff
@@ -335,11 +341,22 @@ std::string RayLog::GetLogFormatPattern() { return log_format_pattern_; }
 
 std::string RayLog::GetLoggerName() { return logger_name_; }
 
+void RayLog::SetExposeLogCallback(const ExposeLogCallback &expose_log_callback) {
+  expose_log_callback_ = expose_log_callback;
```

Comment:
Make it accept a vector of callbacks instead?

Reply:
Make sense.